File: 14-day/primesNumbers.js
function isPrime(number) {
  for (let divider = 2; divider < Math.floor(Math.sqrt(number)) + 1; divider++) {
    if (number % divider == 0) {
      return false
    }
  }
  return true
}

function primeNumbers(limit) {
  let primes = []
  for (let number = 2; number <= limit; number++) {
    if (isPrime(number)) {
      primes.push(number)
    }
  }
  return primes
}

do {
  var limit = parseInt(prompt('Limit number to know primes? '))
  if (limit < 2) alert('Should enter a number greater than or equal to 2')
} while (limit < 2);

alert(`Primes numbers lower than ${limit}: ${primeNumbers(limit)}`)

File: 14-day/primesNumbers.py
def isPrime(number):
    for divider in range(2, int(number ** .5) + 1):
        if (number % divider == 0):
            return False
    return True


def primeNumbers(limit):
    primes = []
    for number in range(2, limit + 1):
        if isPrime(number): 
            primes.append(number)
    return primes


n = int(input('Limit number to know primes? '))
while (n < 2):
    print('Should enter a number greater than or equal to 2')
    n = int(input('Limit number to know primes? '))

print(primeNumbers(n))